Doc/library/argparse.rst

```diff
@@ -1853,6 +1855,13 @@ Argument groups
 
        --bar BAR  bar help
 
+   The optional parameters *argument_default* and *conflict_handler* allow
+   for finer-grained control of the behavior of the argument group. Both
+   of these parameters behave similarly to *argument_default* and
+   *conflict_handler* parameters of :meth:`~ArgumentParser.add_argument`.
+   However, when passes into :meth:`add_argument_group`, these parameters apply
+   to all arguments added to the group.
```

Comment:
I think that you could use references `argument_default_` and `conflict_handler_` here. `add_argument()` does not have such parameters, they are parameters of the parser constructor. Also, it would be better to shorten this description. Just mention that they have the same meaning, but applied to more narrow scope. Also, mention that that parameters are keyword-only.